.pipelines/.vsts-vhd-windows-automation.yaml

```diff
@@ -11,8 +11,25 @@ parameters:
 - name: GithubUserName
   displayName: The user name of the owner to push changes
   type: string
+- name: ReleaseNotes
+  displayName: Release Notes
+  type: boolean
+  default: false
 
 steps:
+- bash: |
+        echo "Removing existing go environment"
+        sudo rm -r /usr/local/go
+        GOLANG_VERSION=1.20.6
```

Comment:
what is the go version installed by default? Could we use the default go or use a different vm image with the required go version?
https://learn.microsoft.com/en-us/azure/devops/pipelines/ecosystems/go?view=azure-devops&tabs=go-current#set-up-go